lib/asar/fs-wrapper.ts

```diff
@@ -646,7 +646,7 @@ export const wrapFsWithAsar = (fs: Record<string, any>) => {
       return;
     }
 
-    if (options.withFileTypes) {
+    if (options?.withFileTypes) {
```

Comment:
looks like our current setup doesn't support this syntax.
```suggestion
    if (options && options.withFileTypes) {
```

Reply:
resolved in rebase